src/server/game/Entities/Player/Player.cpp

```diff
@@ -8575,7 +8574,7 @@ void Player::SendLoot(ObjectGuid guid, LootType loot_type)
                     loot->FillLoot(PLAYER_CORPSE_LOOT_ENTRY, LootTemplates_Creature, this, true);
             }
             // For wintergrasp Quests
-            else if (GetZoneId() == AREA_WINTERGRASP)
+            else if (GetZoneId() == 4197)
```

Comment:
maybe keep the old AREA_WINTERGRASP instead of a magic number ?

Reply:
`AREA_WINTERGRASP` is defined in BattlefieldWG.h which is now outside the game project. I didn't want to include it back here. Since there is no central location for such defines, I opted for the "lazy" version. Suggestions welcome.

Reply:
SharedDefines.h it is then. This way you can also drop that enum from the script project files and use the shared define instead

Reply:
Done!